newjersey/navigator.business.nj.gov
# feat: [#188135142] Add multiple dwelling registration status check

## Description

Adds multiple dwelling status check registration search



### Ticket



This pull request resolves [#188135142](https://www.pivotaltracker.com/story/show/188135142).

### Approach



### Steps to Test

1. Create a Poppy in the "Real Estate Investing" industry
2. Answer yes to the multiple dwelling non essential question
3. Use address in the QA testing doc to see results
4. (NOTE! No addresses in the DB can get you into the incomplete or cancelled state)



### Notes



## Code author checklist
